### ISSUE TITLE
[Business Approvals Kit - QUESTION] How to create an Approval body

### ISSUE THREAD

**Issue by buckthebug** [microsoft/powercat-business-approvals-kit#161]
### What is your question?

We are used to adding request details in to the approval body (as MD) in our custom solutions. For instance, as an invoice approver, I need to know - Inv. number, amount, VAT, etc.
In BAK, the fields/variables aren't pasted to the approval that approvers will receive. I could pass it as a part of "Additional Information" but you append/prepend it with your data (OOF, ...).
The questions:
- Do you plan to append variables to the body as a table by default?
- Or could data be split to several rows in the body? Let's say - BAK internal messages as one row and the content of "Additional Information" another one.

Thank you.

**Comment by endub-22** [comment]
this is an essential feature, I hope there is an answer!

**Comment by craudales** [comment]
I strongly agree. This is a "must" feature. It cannot be used without this. The approver needs to know what he is approving.
